src/software/geom/ray.h

```diff
@@ -49,16 +49,16 @@ class Ray final
 
     inline Segment toSegment() const
     {
-        return Segment(start, direction);
+        return Segment(start, Point(direction));
```

Comment:
1. Add javadocs for functions in this file 
2. Shouldn't this be 

```suggestion
        return Segment(start, Point(start + direction));
```
And this would have length = 1?

Reply:
1. Maybe we should hold off on javadocs as this will be replaced by new_geom ray
2. Fixed in latest update